### PR TITLE
#88 logout functionality 

### PR DESCRIPTION
**Pull request by Kevin-ishimwe**
#### What does this PR do?
adds logout functionality 
#### Description of Task to be completed?
- adds logout fuction
- revokes the token from local storage

#### How should this be manually tested?
go to http://127.0.0.1:5173/dashboard and press logout

### CODE REVIEW

File: src/components/dashboards/admin/bars/SideBar.jsx [hunk 1]
```diff
@@ -45,6 +46,7 @@ function SideBar({ isOpen }) {
     dispatch(setActiveButton('setting'));
   };
   const handleLogout = () => {
+    localStorage.removeItem('token');
     navigate('/login');
```
Comment (Aimeana100) [inline src/components/dashboards/admin/bars/SideBar.jsx]: @Kevin-ishimwe , as our App is not totally pages authenticated , i have opinion that redirecting a user to homepage is better than enforcing him to login after logout!

**Review by Aimeana100 — DISMISSED** [review]

File: src/components/dashboards/admin/bars/SideBar.jsx (hunk 1 above)
Comment (Kevin-ishimwe) [reply src/components/dashboards/admin/bars/SideBar.jsx]: fixed

**Review by FabCode67 — APPROVED** [review]

**Review by BenoitIT — COMMENTED** [review]
good work @Kevin-ishimwe , your code works almost as expected ,however the current functionality like two-factor-authentication are is actually working,you may check that aswell

**Comment by BenoitIT** [comment]
there are two AuthUser.jsx files in redux/reducers folder you can also remove one of them

**Review by BenoitIT — APPROVED** [review]

**Review by Aimeana100 — APPROVED** [review]
Perfect.

**Review by mansurissa — APPROVED** [review]

**Pull request merged**